### ISSUE TITLE
Installation instructions unclear. Flutter Doctor fails to run "doctor --android-licenses". 

### ISSUE THREAD

**Issue by Kaszanas** [flutter/flutter#83374]
## Steps to Reproduce

1. Current state of the documentation page which outlines the steps to install Flutter along with Android SDK are unclear and might be outdated for Android Studio.
I am specifically attempting to follow the instructions outlined here:
https://flutter.dev/docs/get-started/install/windows#set-up-the-android-emulator

While setting up Android Emulator there is no easily accessible "AVD Manager icon" instead the steps are as follows:
1. Click "Configure menu in the bottom right corner of the Android Studio window".
 ![image](https://user-images.githubusercontent.com/34846245/119577233-818cff00-bdba-11eb-8455-93a78fc1f307.png)

2. Select "AVD Manager".

![image](https://user-images.githubusercontent.com/34846245/119577261-92d60b80-bdba-11eb-9d14-2f99731e172d.png)

3. Another step that I am unable to reproduce from the installation instructions is "Under Emulated Performance, select Hardware - GLES 2.0 to enable hardware acceleration.". This setting in Android Studio is set to "Automatic" and I am unable to change it by hand.

4. Finally going forward to check the completeness of my installation while running the command ```flutter doctor``` to check if the installation is complete I was met with a prompt that not all Android Licenses are set. After completing that step I was prompted with yet another information from flutter doctor that I don't have Android Studio installed. This made me follow the command that is posted as a response to this issue: https://github.com/flutter/flutter/issues/67986

Which is precisely: ```flutter config --android-studio-dir=<path/to/installation>```
This has successfully set my Android Studio installation but reversed the correctness of my Android Licenses (?)

5. After that during my attempt to set the Android Licenses once again with the command that is supplied by the flutter doctor I am met with errors that are outlined in the Logs section below.

## Logs

```
PS D:\Projects\SC2MapLocaleExtractor> flutter doctor
Doctor summary (to see all details, run flutter doctor -v):
[√] Flutter (Channel stable, 2.2.0, on Microsoft Windows [Version 10.0.19041.928], locale en-US)
[!] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
    X Android license status unknown.
      Run `flutter doctor --android-licenses` to accept the SDK licenses.
      See https://flutter.dev/docs/get-started/install/windows#android-setup for more details.
[√] Chrome - develop for the web
[√] Android Studio
[√] Connected device (2 available)

! Doctor found issues in 1 category.
```

```
PS D:\Projects\SC2MapLocaleExtractor> flutter doctor -v
[√] Flutter (Channel stable, 2.2.0, on Microsoft Windows [Version 10.0.19041.928], locale en-US)
    • Flutter version 2.2.0 at D:\Programs\flutter
    • Framework revision b22742018b (11 days ago), 2021-05-14 19:12:57 -0700
    • Engine revision a9d88a4d18
    • Dart version 2.13.0

[!] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
    • Android SDK at D:\Programs\AndroidSDK_SDK
    • Platform android-30, build-tools 30.0.3
    • Java binary at: D:\Programs\AndroidSDK\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 11.0.8+10-b944.6842174)
    X Android license status unknown.
      Run `flutter doctor --android-licenses` to accept the SDK licenses.
      See https://flutter.dev/docs/get-started/install/windows#android-setup for more details.

[√] Chrome - develop for the web
    • Chrome at C:\Program Files (x86)\Google\Chrome\Application\chrome.exe

[√] Android Studio
    • Android Studio at D:\Programs\AndroidSDK
    • Flutter plugin can be installed from:
       https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
       https://plugins.jetbrains.com/plugin/6351-dart
    • android-studio-dir = D:\Programs\AndroidSDK
    • Java version OpenJDK Runtime Environment (build 11.0.8+10-b944.6842174)

[√] Connected device (2 available)
    • Chrome (web) • chrome • web-javascript • Google Chrome 90.0.4430.212
    • Edge (web)   • edge   • web-javascript • Microsoft Edge 90.0.818.62

! Doctor found issues in 1 category.
```

## Additional information

Following the instructions from:
https://developer.android.com/studio/command-line/sdkmanager
https://stackoverflow.com/questions/39760172/you-have-not-accepted-the-license-agreements-of-the-following-sdk-components

```
PS D:\Programs\AndroidSDK_SDK\tools\bin> .\sdkmanager.bat --licenses
Warning: File C:\Users\kasza\.android\repositories.cfg could not be loaded.
All SDK package licenses accepted.======] 100% Computing updates...
```

I have managed to verify that I have all SDK licenses accepted but the question of ```flutter doctor``` being unable to verify that still stands.

**Comment by jonahwilliams** [comment]
Do you have the cmline tools component installed in android studio, or the platform tools (deprecated) component, or both?

**Comment by Kaszanas** [comment]
@jonahwilliams 
Up to my knowledge I was not installing any additional tools that were not provided in the standard Android Studio and Android SDK installation.

I have verified that indeed I did not have Android cmdline-tools installed and I have used these instructions: https://developer.android.com/studio/releases/cmdline-tools to download them 

```
.\sdkmanager.bat 'cmdline-tools;latest' --channel=3
Warning: File C:\Users\kasza\.android\repositories.cfg could not be loaded.
[=======================================] 100% Unzipping... cmdline-tools/lib/RE
```

Up to my knowledge I have the latest stable releases of Flutter, Dart, and Android Studio / Android SDK installed as I have been following the installation instructions directly before posting this issue.

On the other hand it seems that flutter itself works correctly and I am going to test its functionality further.

The licenses seem to be accepted but they fail to be recognized by ```flutter doctor```

**Comment by TahaTesser** [comment]
Hi

> The licenses seem to be accepted but they fail to be recognized by flutter doctor

Can you please provide what error you getting  here?

**Comment by Kaszanas** [comment]
@TahaTesser @jonahwilliams 

It seems that the problem resolved itself (?)
I have no idea how because I have not done anything specific since yesterday and my comment that was posted before.

I have noticed that ```flutter doctor``` had another issue concerning with being unable to run main function and provided logs (from Java) but I have failed to save these yesterday and I am unable to reproduce this issue today.

Feel free to close this issue as it seems to be resolved.

And yes @TahaTesser the issue was that the ```Android license status``` was ```[!]``` and set to unknown even though the licenses were previously accepted.
